sqlobject/src/main/java/org/jdbi/v3/sqlobject/statement/ResultReturner.java

```diff
@@ -177,7 +179,7 @@ protected Type elementType(StatementContext ctx)
         @Override
         protected Object result(ResultIterable<?> bearer, StatementContext ctx)
         {
-            return bearer.findFirst().orElse(null);
+            return bearer.findFirst().map(Object.class::cast).orElseGet(() -> DefaultResult.forType(returnType));
```

Comment:
Java compiler barfs without `.map(Object.class::cast)`, since it can't convert `capture of ?` to `Object`.

Bleh.